### ISSUE TITLE
Provide an online demo page.

### ISSUE THREAD

**Issue by pranavrajs** [lonesword/openmosaic#1]
Use gh-pages

**Comment by kevinmartinjos** [comment]
Good point. The "demo" would need a server. I'm thinking more like a page with some GIFs

**Comment by pranavrajs** [comment]
@lonesword Heroku is a nice option.

**Comment by avinayak** [comment]
ayee

**Comment by kevinmartinjos** [comment]
Mindi povaruthu. Asked you to commit the demo example we presented an year ago. Is it still there on your hard disk?

**Comment by avinayak** [comment]
atu poyi .. njan ninte lapila demo undakkiye :laughing:
